ocean-client/src/programs/lm-reinvest-program.ts

```diff
@@ -107,10 +107,13 @@ export class LMReinvestProgram extends CommonProgram {
   }
 
   async checkAndDoReinvest(balances: Map<string, AddressToken>, telegram: Telegram): Promise<boolean> {
-    const maxReinvestForDonation = Math.max(this.getSettings().reinvestThreshold!, 20) * 2 //anything below 20 DFI is considered a "reinvest all the time"
+    const maxReinvestThreshold = Math.max(
+      this.getSettings().reinvestThreshold! * 2,
+      +(process.env.VAULTMAXI_MAXREINVEST ?? 40),
+    ) //anything below 20 DFI is considered a "reinvest all the time"
```

Comment:
why not fallback to 0 if the env variable isn't set?

Reply:
it was `max(threshold,20)*2` . This was needed for users who set the reinvestThreshold too low, so it accumulates more than the threshold within 15 minutes. In vault-maxi we can estimate the rewards per 15 minutes from the LMPair, here its harder.

This way of writing is just a shortcut for `max(max(threshold,20)*2 , VAULTMAXI_MAXREINVEST ?? 0)`